src/main_minimal.cpp

```diff
@@ -305,8 +311,8 @@ void tick_all_systems(const SysTick_s &current_system_tick)
         steering_system.getSteeringSystemData(),
         a2.get().conversions[MCU15_FL_LOADCELL_CHANNEL], // FL load cell reading. TODO: fix index
         a3.get().conversions[MCU15_FR_LOADCELL_CHANNEL], // FR load cell reading. TODO: fix index
-        (const AnalogConversion_s){},                    // RL load cell reading. TODO: get data from rear load cells
-        (const AnalogConversion_s){},                    // RR load cell reading. TODO: get data from rear load cells
+        sab_interface.rlLoadCell.convert(),  // RL load cell reading. TODO: get data from rear load cells
```

Comment:
AnalogChannel only returns AnalogConversion_s when .convert(), they don't store it themselves. It probably doesn't hurt but those convert()'s in SABInterface when called upon dispatching CAN messages seems not to make a difference (?)
<img width="501" alt="image" src="https://github.com/hytech-racing/MCU/assets/113149455/522b9c9a-cc6f-407f-8828-5d7870e3bd9d">
<img width="673" alt="image" src="https://github.com/hytech-racing/MCU/assets/113149455/cb30801a-c992-4a8c-af1d-1b160a5c2af8">

But if SAB has a return struct that consists of conversions would make sense to have them. Not strictly necessary though.